### PR TITLE
Bug 1858872 - Switch "windows" with "tabs" in private mode description

### PR DESCRIPTION
**Pull request by HarrisonOg**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1858872

### CODE REVIEW

File: fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@
         private mode in our new Total Private Browsing mode.
         The first parameter is the name of the app defined in app_name (for example: Firefox Nightly)
         The second parameter is the clickable link text in felt_privacy_info_card_subtitle_link_text -->
-    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. %2$s</string>
+    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private tabs. %2$s</string>
```
Comment (gabrielluong) [inline fenix/app/src/main/res/values/strings.xml:75]: This string was added in 119 [(Bug 1847819)](https://bugzilla.mozilla.org/show_bug.cgi?id=1847819). We can't simply change the string directly anymore. We need to follow the process of adding a new string ID and marking the old one for removal if it's unused. See https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/docs/Working-with-Strings.md

File: fenix/app/src/main/res/values-en-rCA/strings.xml [hunk 2]
```diff
@@ -73,7 +73,7 @@
         private mode in our new Total Private Browsing mode.
         The first parameter is the name of the app defined in app_name (for example: Firefox Nightly)
         The second parameter is the clickable link text in felt_privacy_info_card_subtitle_link_text -->
-    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. %2$s</string>
+    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private tabs. %2$s</string>
```
Comment (gabrielluong) [inline fenix/app/src/main/res/values-en-rCA/strings.xml:76]: We should only ever be changing the values in `fenix/app/src/main/res/values/strings.xml`. These files that are imported from Pontoon (our localization service) and should never be manually edited.

**Review by boek — DISMISSED** [review]
Like GL pointed out. We'll have to create a new string and keep this one around

File: fenix/app/src/main/res/values-en-rCA/strings.xml (hunk 2 above)
Comment (HarrisonOg) [reply fenix/app/src/main/res/values-en-rCA/strings.xml]: Reset the changes to the translated files.

File: fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (HarrisonOg) [reply fenix/app/src/main/res/values/strings.xml]: Ah I see, I created a new string and relinked the info card text to that new string.  Marked the old one for removal as well.

File: fenix/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -72,7 +72,8 @@
         private mode in our new Total Private Browsing mode.
         The first parameter is the name of the app defined in app_name (for example: Firefox Nightly)
         The second parameter is the clickable link text in felt_privacy_info_card_subtitle_link_text -->
-    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. %2$s</string>
+    <string name="felt_privacy_info_card_subtitle" moz:removedIn="120" tools:ignore="UnusedResources">%1$s deletes your cookies, history, and site data when you close all your private windows. %2$s</string>
+    <string name="felt_privacy_info_card_subtitle_two">%1$s deletes your cookies, history, and site data when you close all your private tabs. %2$s</string>
```
Comment (gabrielluong) [inline fenix/app/src/main/res/values/strings.xml:76]:
```suggestion
    <string name="felt_privacy_info_card_subtitle_2">%1$s deletes your cookies, history, and site data when you close all your private tabs. %2$s</string>
```
This also needs a localization comment. Could you reuse the one from above?

Comment (HarrisonOg) [reply fenix/app/src/main/res/values/strings.xml:76]: ok, adding it.

**Review by gabrielluong — APPROVED** [review]

**Pull request merged**